api/src/utils/validator.js

```diff
@@ -611,6 +611,105 @@ function validateSelf(referent) {
     .validate(referent, { stripUnknown: true });
 }
 
+const validateInscription = (value) => {
+  return Joi.object()
+    .keys({
+      email: Joi.string().allow(null, ""),
```

Comment:
```suggestion
      email: Joi.string().lowercase().trim().email().allow(null, ""),
```

Plus globalement pourquoi on remettrait pas tous les contrôles qu'on a mis ailleurs ?